apps/heureka/src/components/Services/utils.ts

```diff
@@ -39,35 +40,60 @@ type NormalizedServices = {
   services: ServiceType[]
 }
 
-export const getNormalizedData = (data: GetServicesQuery | undefined): NormalizedServices => ({
-  totalCount: data?.Services?.totalCount || 0,
-  pages: data?.Services?.pageInfo?.pages?.filter((edge) => edge !== null) || [],
-  services: isNil(data?.Services?.edges)
-    ? []
-    : data?.Services?.edges
-        ?.filter((edge) => edge !== null)
-        .map((edge?: Edge): ServiceType => {
-          const node: Service | undefined = edge?.node
-          const service: ServiceType = {
-            id: node?.id?.toString() || "",
-            name: node?.ccrn || "",
-            serviceDetails: {
-              supportGroups: getSupportGroups(edge),
-            },
-            components: node?.objectMetadata?.componentInstanceCount || 0,
-            serviceOwners: getServiceOwners(edge),
-            issuesCount: {
-              critical: Math.floor(Math.random() * 10), //TODO: remove mock data when available
-              high: Math.floor(Math.random() * 10), //TODO: remove mock data when available
-            },
-            remediationDate: "2023-01-01", //TODO: remove mock data when available
-          }
-          return service
-        }),
-})
+type ExtendedService = Service & {
```

Comment:
I'm thinking shouldn't we add those files to the `ServiceType` instead because this information belongs to `Service`.

Reply:
This need discussion

Reply:
discussed and we keep it like for now